tox.ini

```diff
@@ -30,7 +30,7 @@ commands = pytest {posargs: -p no:warnings tests}
 # Bottom of file has Flake8 settings
 commands = flake8 {posargs:application.py config consoler.py damien scripts tests mrsbaylock}
 deps =
-    flake8>=4.0.1
+    flake8==4.0.1
```

Comment:
@pauline2k flake8 5.0 dropped over the weekend, and they seem to still be working out some bugs.

Reply:
Culprit apprehended!